resources/metric-ui.mjs

```diff
@@ -239,8 +239,12 @@ function renderScatterPlot({ values, width = 500, height, trackHeight, xAxisPosi
     // Axis + labels height:
     const axisHeight = 18;
     const axisMarginY = 4;
-    const markerSize = 5;
     const trackMargin = 2;
+    let markerSize = 5;
+    // Auto-adjust markers to [2px, 5px] for high iteration counts:
+    const iterationsLimit = 20;
+    if (values.length > iterationsLimit)
+        markerSize = 1 + (4 / values.length) * iterationsLimit;
```

Comment:
I don't mind, but what do you think of a simpler:
```js
const markerSIze = values.length <= 20 ? 5 : 2; // adjust the marker size to the iteraction count
```
But if you prefer your formula that's also fine by me.

Reply:
The formula seems easy enough IMO (compared the surrounding complexity :P)